trunk/bin/ingesttar.py

```diff
@@ -10,15 +10,20 @@
 _dir = os.environ['LCOSNDIR']
 
 def ingesttar(_tarfile,force=False):
-    _targetid = re.sub('.tar.gz','',string.split(_tarfile,'_')[-1])
+    if 'gz' in _tarfile:
+        _targetid = re.sub('.tar.gz','',string.split(_tarfile,'_')[-1])
+    else:
+        _targetid = re.sub('.tar','',string.split(_tarfile,'_')[-1])
     my_tar = tarfile.open(_tarfile)
     imglist = my_tar.getnames()
```

Comment:
This would be better as 
imglist = [os.path.join(_dir, i) for i in my_tar.getnames()]
rather than defining a totally new variable.